### PR TITLE
Add base structure for console app

### PR DESCRIPTION
**Pull request by hihuz**
This is the first PR for the API console, setting up the app structure and base authentication with the Kontist SDK.

Couple points for discussion:
* I used create-react-app to speed up the initial setup, I think it is fine but I might have to eject at some point because it is quite restrictive
* directory for the source of the app is `console-src` I don't like it so much but couldn't come up with something better 🤔 
* I will submit my PRs against a feature branch until the whole feature is ready to go

if you want to test locally, `cd console-src && yarn && yarn start`

### CODE REVIEW

File: console-src/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (hihuz) [inline console-src/.gitignore:1]: Maybe I should put that in the root gitignore file 🤔

**Comment by johannespfeiffer** [comment]
Looks good so far 👍

**Review by sgalonska — COMMENTED** [review]
this too big of a pull request for me to review. Please check with @johannespfeiffer

**Comment by hihuz** [comment]
@sgalonska do you mean because of the large number of changes?

Most of these are auto generated files, to setup the base app structure. They are not supposed to be reviewed.

There are at most 200 changes to review in this PR, I don't think it makes sense to reduce it further in this case

**Comment by johannespfeiffer** [comment]
Agree, this initial PR contains autogenerated files and static assets that need to be in this repo as we use GitHub pages to directly publish them. Further PRs will be smaller and only contain relevant code.

**Review by johannespfeiffer — APPROVED** [review]

**Pull request merged**